index.html

```diff
@@ -35,6 +35,12 @@ <h2>Hauptleitung</h2>
 <p><a class="ellipsis" href="mailto:matthias.niklaus@ref-sh.ch">Matthias Niklaus</a></p>
 <p><a class="ellipsis" href="mailto:philippe.niklaus@ref-sh.ch">Philippe Niklaus</a></p>
 
+<h2>Sponsoren</h2>
+<p class="trim">Unser Sola wird Unterstüzt von:</p>
```

Comment:
`SOLA` is written in all caps, and a typo in `unterstützt`

```suggestion
<p class="trim">Unser SOLA wird unterstützt von:</p>
```